### ISSUE TITLE
Request to know Snap status

### ISSUE THREAD

**Issue by jameschensmith** [tunix/alacritty-snap#1]
Hey @tunix :wave::blush:

What's the status of this snap? It looks great so far! I can see possibly a few more things to add before it's added to the Snapcrafters' GitHub organization & published on Snapcraft. If you need any help, create a few issues and let me know. I'd be happy to help in any way that I can!

**Comment by tunix** [comment]
Hey @jameschensmith ,

Thanks for your message. There is a thread where I'm trying to move the snap to snapcrafter's administration. There is an outdated version inside the Snap Store and I wanted to claim it. This is the solution they offered. I'm OK for moving it into snapcrafter's repo but they have been really slow for merging my PR's etc -- so that's one concern.

There are a few issues with this snap which I'm not able to solve and most of them started after I've upgraded from 20.04 to 20.10:

* I cannot get transparency to work.
* Screenshots are empty white squares when in full screen mode

I actually need help to iron out these issues but I don't know to whom I should talk to. I've read most of the issues in alacritty's github repo but couldn't find any pointers.

**Comment by jameschensmith** [comment]
> There is a thread where I'm trying to move the snap to snapcrafter's administration. There is an outdated version inside the Snap Store and I wanted to claim it. This is the solution they offered.

Yeah, I've seen those. I was thinking you could reserve `alacritty-terminal-emulator`, `alacritty-terminal`, or something else for now until that name opens. What are your thoughts on that?

> I'm OK for moving it into snapcrafter's repo but they have been really slow for merging my PR's etc -- so that's one concern.

Oof. I thought by "Join Snapcrafters", they would bring your project in with you still as the maintainer. Is this not the case? That does sound concerning :confused:

> There are a few issues with this snap which I'm not able to solve and most of them started after I've upgraded from 20.04 to 20.10:
> 
> * I cannot get transparency to work.
> * Screenshots are empty white squares when in full screen mode
> 
> I actually need help to iron out these issues but I don't know to whom I should talk to. I've read most of the issues in alacritty's github repo but couldn't find any pointers.

Let's get issues created here to track those two (and any other) issues you have. That way we can make the concerns visible. It's also good for linking in case the issue is indeed on the side of the application vs. the packaging. I'll start looking into it, but since you have more context of the issues, it would be good for you to create the initial issues. Sound good? :slightly_smiling_face:

**Comment by tunix** [comment]
> > There is a thread where I'm trying to move the snap to snapcrafter's administration. There is an outdated version inside the Snap Store and I wanted to claim it. This is the solution they offered.
> 
> Yeah, I've seen those. I was thinking you could reserve `alacritty-terminal-emulator`, `alacritty-terminal`, or something else for now until that name opens. What are your thoughts on that?

You're a better namer than me! 😄  I actually registered alacritty-tunix as it's kind of a convention by now but since this being a classic snap, I've to go through the manual reviewing process. But yeah, we can definitely name it as you suggest.

> 
> > I'm OK for moving it into snapcrafter's repo but they have been really slow for merging my PR's etc -- so that's one concern.
> 
> Oof. I thought by "Join Snapcrafters", they would bring your project in with you still as the maintainer. Is this not the case? That does sound concerning confused

That's exactly what I initially thought when I submitted my snaps. (gradle first, now alacritty) However it seems to be not the way that is. I still have access to the snap on snapcraft.io but they own the repo. So they're the gatekeepers for new packages, fixes to existing ones etc. and so far it's been disappointingly slow.

> 
> > There are a few issues with this snap which I'm not able to solve and most of them started after I've upgraded from 20.04 to 20.10:
> > 
> > * I cannot get transparency to work.
> > * Screenshots are empty white squares when in full screen mode
> > 
> > I actually need help to iron out these issues but I don't know to whom I should talk to. I've read most of the issues in alacritty's github repo but couldn't find any pointers.
> 
> Let's get issues created here to track those two (and any other) issues you have. That way we can make the concerns visible. It's also good for linking in case the issue is indeed on the side of the application vs. the packaging. I'll start looking into it, but since you have more context of the issues, it would be good for you to create the initial issues. Sound good? slightly_smiling_face

You mean creating issues in this repo, right? Actually, that's a great idea as well! 😄 I'll create them later today. Thanks!

**Comment by jameschensmith** [comment]
> > > I'm OK for moving it into snapcrafter's repo but they have been really slow for merging my PR's etc -- so that's one concern.
> > 
> > 
> > Oof. I thought by "Join Snapcrafters", they would bring your project in with you still as the maintainer. Is this not the case? That does sound concerning confused
> 
> That's exactly what I initially thought when I submitted my snaps. (gradle first, now alacritty) However it seems to be not the way that is. I still have access to the snap on snapcraft.io but they own the repo. So they're the gatekeepers for new packages, fixes to existing ones etc. and so far it's been disappointingly slow.

Just an update on looking into this. It appears [Alan Pope recently just left Canonical](https://www.reddit.com/r/linux/comments/mn0bzs/alan_pope_ubuntu_developer_leaves_canonical/). I know that he had a large role in Snapcrafters. There's a new thread in the Snapcraft forum on [who will run Snapcrafters](https://forum.snapcraft.io/t/who-will-run-snapcrafters/23802). Keeping a close eye on this could provide a good answer to the next step for the Alacritty snap with regards to being added to the Snapcrafters' organization. :slightly_smiling_face: :+1:

**Comment by tunix** [comment]
Thanks @jameschensmith , I just commented on that thread. I hope all will be sorted out soon.

**Comment by tunix** [comment]
I just created the known issues. I don't know what else I can add for the context of those issues. It's been a while since I dealt with them.

**Comment by tunix** [comment]
@jameschensmith - is the cursor size issue solved for you with the latest changes? If not, could you please create an issue for that? Currently I'm not aware of any other problem which is awesome! 🎉

**Comment by jameschensmith** [comment]
> @jameschensmith - is the cursor size issue solved for you with the latest changes? If not, could you please create an issue for that? Currently I'm not aware of any other problem which is awesome!  The only thing left to do is to get this snap on the store! 🤞🏻

Not yet. I've created #6 to look into. We may also need to add some additional info into `snapcraft.yaml` (such as snap summary, description, license, etc.) before it gets published to the snap store, but that won't be difficult :slightly_smiling_face: Let's get it published! :rocket:

**Comment by tunix** [comment]
Good news! We had [multiple](https://forum.snapcraft.io/t/who-will-run-snapcrafters/23802/5) [discussions](https://forum.snapcraft.io/t/request-for-classic-confinement-alacritty-alacritty-tunix/21672/21) with different people and as of now, alacritty snap is transferred to my account. 🎉 

I'll start publish asap.

**Comment by tunix** [comment]
As of now, we're on Snap Store with latest 0.7.2 release! 🚀

**Comment by jameschensmith** [comment]
Awesome news! :clap: :grin: Thank you for your persistence on this Snap :slightly_smiling_face: 

We should get a pipeline started! If not for publishing (can classic snaps publish from a pipeline?), then for ensuring more stable builds.

Also, I've posted a snapshot below of the how the current Alacritty snap looks in the Snapcraft store. Is the information from our repository, or was it from the previous owner's repository? If it's from the latter, we should get this metadata in our repository.

Lastly, what are your thoughts on closing this issue? Do you think this issue should track the status of a future transfer (as you noted from your multiple discussions), or should we open a new issue to track that future view?

![image](https://user-images.githubusercontent.com/15643597/116634585-19b1d880-a922-11eb-8301-283e09f6c172.png)

**Comment by tunix** [comment]
> We should get a pipeline started! If not for publishing (can classic snaps publish from a pipeline?), then for ensuring more stable builds.
> 
> Also, I've posted a snapshot below of the how the current Alacritty snap looks in the Snapcraft store. Is the information from our repository, or was it from the previous owner's repository? If it's from the latter, we should get this metadata in our repository.

I've updated all information on that page and everything comes from our snapcraft.yml and specificially from the metadata file inside alacritty's repository. (snap kind of proxies the information from an XML inside alacritty repo) Also I've uploaded the official screenshot that I found in alacritty's repository. :)

> Lastly, what are your thoughts on closing this issue? Do you think this issue should track the status of a future transfer (as you noted from your multiple discussions), or should we open a new issue to track that future view?

I think we can close this issue for now. This package will eventually be transferred to snapcrafters repository where I, amoung many other contributors, will have access to.
